### PR TITLE
Add unique project capability to workload SDK

### PR DESCRIPTION
**Pull request by danegsta**
Adding a unique project capability to the Aspire.Hosting.Sdk workload SDK to allow tooling to identify Aspire projects before project restore.

### CODE REVIEW

File: src/Aspire.Hosting.Sdk/SDK/Sdk.targets [hunk 1]
```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <ProjectCapability Include="DynamicFileNesting" />
     <ProjectCapability Include="DynamicFileNestingEnabled" />
+    <ProjectCapability Include="AspireOrchestration" />
```
Comment (vijayrkn) [inline src/Aspire.Hosting.Sdk/SDK/Sdk.targets:14]: Wouldn't it be better to call it "AspireAppHost" to match the template naming convention?

Comment (DamianEdwards) [reply src/Aspire.Hosting.Sdk/SDK/Sdk.targets:14]: I think the thought process here is that it's added by the workload SDK which itself is added to the project by the `IsAspireHost` property in the project, and the workload SDK only exists to facilitate the delivery of the orchestration pieces (DCP) so calling it `AsprieOrchestration` is inline with that and separate from the `IsAspireHost` property.

Comment (danegsta) [reply src/Aspire.Hosting.Sdk/SDK/Sdk.targets:14]: That was my main thought in using `AspireOrchestration` vs `AspireHost`, `AspireAppHost`, etc.; it seems as though the existing `Aspire` capability declared in the `Aspire.Hosting` targets is basically the AppHost capability, so having a distinct enough capability for the workload SDK made sense.

**Review by BillHiebert — APPROVED** [review]

File: src/Aspire.Hosting.Sdk/SDK/Sdk.targets (hunk 1 above)
[the remark this answers is not in the document]
Comment (danegsta) [reply src/Aspire.Hosting.Sdk/SDK/Sdk.targets:14]: If the tooling isn't tightly coupled to the `Aspire.Hosting` assembly being restored, you can rely on this before restore completes. If you do have something that's tied to `Aspire.Hosting` in some way, the existing `Aspire` capability is probably the safer option in that case.

[the remark this answers is not in the document]
Comment (vijayrkn) [reply src/Aspire.Hosting.Sdk/SDK/Sdk.targets:14]: @phenning - For the readme support, can we switch to this capability?

**Pull request merged**